app/upload/evaporate.d.ts

```diff
@@ -0,0 +1,9 @@
+declare class Evaporate {
```

Comment:
yeah, I have submitted it as a PR:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/8742